charter.md

```diff
@@ -85,15 +93,123 @@ are replacing, regardless of the length of that remainder.
 
 [voter-eligibility]: /elections.md#eligibility-for-voting
 
-## Quorum
+### Resignation
+
+If a committee member chooses not to continue in their role, for whatever
+self-elected reason, they must notify the committee in writing.
+
+### Removal
+
+#### No confidence
+
+A Steering Committee member may be removed by an affirmative vote of a
+**_three-quarters supermajority of the
+[fixed membership of the committee](#composition)_**.
+
+Example:
+
+* 7 (members) / 4 = 1.75
+* 1.75 * 3 = 5.25
+* Round up to the nearest whole number (6)
+* Six (6) affirmative votes would be required to remove a member through a vote
+  of no confidence
+
+The call for a vote of no confidence will happen in a public Steering Committee meeting and a
+GitHub issue in kubernetes/steering by a current member and must be seconded by
```

Comment:
"kubernetes/steering" -> "kubernetes/steering repository"

Reply:
perhaps also make "kubernetes/steering repository" a self-link to the repo here for clarity that doesn't lengthen the rendered document.

Reply:
Fixed in https://github.com/kubernetes/steering/pull/248/commits/107673f27720e863a7c2f0c460eafe3f98a7e2d6.